pom.xml

```diff
@@ -1510,6 +1510,7 @@
         <findbugs.skip>true</findbugs.skip>
         <xml.skip>true</xml.skip>
         <forbiddenapis.skip>true</forbiddenapis.skip>
+        <jacoco.skip>true</jacoco.skip>
```

Comment:
jacoco is defined in special profile, other builds should not be affected.
please remove.

Reply:
same